tests/e2e/vcp_to_csi_syncer.go

```diff
@@ -978,6 +987,319 @@ var _ = ginkgo.Describe("[csi-vcp-mig] VCP to CSI migration syncer tests", func(
+		Steps:
+		1.	Create VCP SC SC1
+		2.	Create a namespace NS1
+		3.	Create 10 statefulsets with 5 PVCs each using SC1 wait for all replicas in each of the statefulset to reach 'Running` state
+		4.	Enable CSIMigration and CSIMigrationvSphere feature gates on kube-controller-manager (& restart)
+		5.	Repeat the following steps for all the nodes in the k8s cluster
+			a.	Drain and Cordon off the node
+			b.	Enable CSIMigration and CSIMigrationvSphere feature gates on the kubelet and Restart kubelet.
+			c.	verify CSI node for the corresponding K8s node has the following annotation - storage.alpha.kubernetes.io/migrated-plugins
+			d.	Enable scheduling on the node
+		6.	wait for all PVs and PVCs to get the following annotation -  "pv.kubernetes.io/migrated-to": "csi.vsphere.vmware.com"
+		7.	Verify cnsvspherevolumemigrations crd is created for all the migrated volumes
+		8.	wait for all replicas in each of the statefulset from step 3 to reach 'Running` state
+		9.	Create 10 statefulsets with 5 PVCs each using SC1 wait for all replicas in each of the statefulset to reach 'Running` state
+		10.	wait for all PVs and PVCs  created in step 9 to get the following annotation -  "pv.kubernetes.io/provisioned-by": "csi.vsphere.vmware.com"
+		11.	Verify cnsvspherevolumemigrations crd is created for all the migrated volumes
+		12.	verify CNS entries for all 100 PV/PVCs (and their entity references)
+		13.	delete 5 statefulsets each which were created in step 3 and 9
+		14.	verify CNS entries for all 50 PVCs (and their entity references) which were affected by step 13
+		15.	Delete PVCs associated with statefulsets deleted in step 13
+		16.	Wait and verify CNS entries for deleted items are gone
+		17.	delete namespace NS1
+		18.	Wait and verify CNS entries for deleted items are gone
+		19.	Delete SC1
+		20.	Repeat the following steps for all the nodes in the k8s cluster
+			a.	Drain and Cordon off the node
+			b.	Disable CSIMigration and CSIMigrationvSphere feature gates on the kubelet and Restart kubelet.
+			c.	verify CSI node for the corresponding K8s node does not have the following annotation - storage.alpha.kubernetes.io/migrated-plugins
+			d.	Enable scheduling on the node
+		21.	Disable CSIMigration and CSIMigrationvSphere feature gates on kube-controller-manager (& restart)
+	*/
+	ginkgo.It("vcp to csi mini feature stress", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		ns, err := framework.CreateTestingNS(f.BaseName, client, nil)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		ginkgo.By("Creating VCP SC")
+		scParams := make(map[string]string)
+		scParams[vcpScParamDatastoreName] = GetAndExpectStringEnvVar(envSharedDatastoreName)
+		vcpSc, err := createVcpStorageClass(client, scParams, nil, "", "", false, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		vcpScs = append(vcpScs, vcpSc)
+
+		stss := []*appsv1.StatefulSet{}
+		fcdIds := []string{}
+		for i := 0; i < 10; i++ {
```

Comment:
can you add a comment here, like `ginkgo.By("Creating 10 statefulset pods with 5 replicas")`

Reply:
it is mentioned in the test steps